admin/src/main/scala/io/buoyant/admin/Admin.scala

```diff
@@ -114,7 +115,7 @@ class Admin(val address: SocketAddress) {
         log.debug(s"admin: $url => ${service.getClass.getName}")
         muxer.withHandler(url, service)
     }
-    notFoundView.andThen(muxer)
+    HeadFilter andThen notFoundView andThen muxer
```

Comment:
Also good to know, thanks; I'll keep that in mind. I wasn't sure about coding style, all I saw was "use Effective Scala"